FFM-7010 Add Custom TLS Example (#3798)
harness/developer-hub: docs/feature-flags/ff-sdks/server-sdks/feature-flag-sdks-go-application.md

| streamEnabled | `harness.WithStreamEnabled(false)` | Set to `true` to enable streaming mode.Set to `false` to disable streaming mode. | `true` |
| analyticsEnabled | `harness.WithAnalyticsEnabled(false)` | Set to `true` to enable analytics.Set to `false` to disable analytics.**Note**: Analytics are not cached. | `true` |

> Enabling analytics is currently not available using the Go SDK.For further configuration options and samples, such as configuring your logger or using the SDK with the Relay Proxy, go to [Additional Options](feature-flag-sdks-go-application.md#additional-options).

Author: Just outdated. Go does support metrics now

harness.WithURL("http://localhost:7000"),  
harness.WithEventsURL("http://localhost:7000"))
```

### Configure your HTTP Client

> The SDK has a default HTTP client, however, you can provide your own HTTP client and passing it is as a configuration option.

Author: ```suggestion
The SDK has a default HTTP client, however, you can provide your own HTTP client to the SDK by passing it in as a configuration option.
```
